src/security.tf

```diff
@@ -70,7 +70,6 @@ data "azuread_group" "adgroup_developers" {
 
 ## ad group policy ##
 resource "azurerm_key_vault_access_policy" "adgroup_developers_policy" {
-  count = var.env_short == "d" ? 1 : 0
```

Comment:
in this way any developer could be change prod kv too.
Are we sure it's right ? Or only sit and uat.
IMHO this give us many power but new developer too, could by mistake (ex. not paying attention to environment ) change the configuration in production.

Reply:
@pasqualespica  I've assigned you and Simone the owner role to all the subscriptions.
This PR is no more needed.